pyproject.toml

```diff
@@ -48,4 +48,4 @@ ignore_missing_imports = true
 allow_redefinition = true
 
 [tool.pylint.'MESSAGES CONTROL']
-disable = "too-few-public-methods,too-many-arguments,too-many-lines,line-too-long,fixme"
+disable = "too-few-public-methods,too-many-arguments,too-many-lines,line-too-long,too-many-locals,fixme,duplicate-code"
```

Comment:
What is the reason for adding these? Is it on test or source files?

Reply:
`duplicate-code` is not needed anymore, I removed that. `too-many-locals` is in `tests/integration/test_upgrade.py`.

Reply:
Would it be better to specifically add it to the files where it needs to be disabled? E.g., just put `# pylint: disable=too-many-locals` at the top of the file and it will be disabled for the whole file. This way the check is disabled for all files

Reply:
Updated, thanks!